modules/exploits/windows/iis/iis_webdav_ScStoragePathFromUrl.rb

```diff
@@ -0,0 +1,77 @@
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GoodRanking
+
+  include Msf::Exploit::Remote::Tcp
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'CVE-2017-7269 Microsoft IIS WebDav ScStoragePathFromUrl Overflow',
+      'Description'    => %q{
+          Buffer overflow in the ScStoragePathFromUrl function in the WebDAV service in Internet Information Services (IIS) 6.0 in Microsoft Windows Server 2003 R2 allows remote attackers to execute arbitrary code via a long header beginning with "If: <http://" in a PROPFIND request, as exploited in the wild in July or August 2016.
+          Original exploit by Zhiniang Peng and Chen Wu.
+      },
+      'Author'         => [ 'Dominic Chell <dominic@mdsec.co.uk>' ],
```

Comment:
In cases where there is an "Original Author" they generally each get their own lines within the module. You can then specify who did what by using comments like:
```
'Zhiniang Peng',  # original exploit
'Dominic Chell <dominic@mdsec.co.uk>' # metasploit module
```